images/win/scripts/Installers/Install-PostgreSQL.ps1

```diff
@@ -0,0 +1,59 @@
+function Install-PostgreSQL {
+    $installPath= choco install postgresql | Select-String "Installed to"
```

Comment:
Is there any better way to get postgresql path? I don't like to rely on output logs.

Reply:
Probably `which psql`?

Reply:
Please see Notes section: https://chocolatey.org/packages/postgresql